corelight_md.md

```diff
@@ -1,12 +1,12 @@
 # Collect Corelight Sensor logs
 
-This document describes how you can collect Corelight sensor logs by configuring the Corelight sensor and a Chronicle forwarder. This document also lists the supported log types and Corelight version.
+This document describes how you can collect Corelight sensor logs by configuring the Corelight sensor and a Chronicle forwarder. This document also lists the supported log types and this parser supports all Corelight sensors versions from v27.4 and earlier.
```

Comment:
I would change "This document also lists the supported log types and this parser supports all Corelight sensors versions from v27.4 and earlier." to something like "This document also lists the supported log types and supported Corelight versions."

The specific versions are mentioned later in the document.

Reply:
Updated